vk/vk/View/Cells/GroupTableViewCell.swift

```diff
@@ -13,16 +13,32 @@ final class GroupTableViewCell: UITableViewCell {
     // MARK: - Public methods
 
     func configure(with group: Group) {
-        groupImaveView.image = UIImage(named: group.photoName)
+        updateImageView(with: group)
         groupNameLabel.text = group.name
+        setupGestureRecognizer()
+    }
+
+    // MARK: - Private Methods
+
+    private func updateImageView(with group: Group) {
+        if let imageData = group.imageData {
+            groupImaveView.image = UIImage(data: imageData)
+        } else {
+            groupImaveView.image = nil
+            group.loadImage { [weak self] imageData in
```

Comment:
```suggestion
            group.loadImage { [weak self] imageData in
```
из модели нельзя это вызывать нужно из отдельного сервиса